File: API/src/main/java/com/ticketease/api/Enums/StatusEnum.java
package com.ticketease.api.Enums;

public enum StatusEnum {
    PENDING_APPROVAL("Aguardando Aprovação"),
    NEW("Novo"),
    IN_PROGRESS("Em Andamento"),
    PENDING("Pendente"),
    RESOLVED("Resolvido"),
    CLOSED("Fechado"),
    CANCELED("Cancelado");

    private final String label;

    StatusEnum(String label) {
        this.label = label;
    }

    public String getLabel() {
        return label;
    }
}

File: frontend/src/enums/StatusEnum.ts
export enum StatusEnum {
	PENDING_APPROVAL = "PENDING_APPROVAL",
	NEW = "NEW",
	IN_PROGRESS = "IN_PROGRESS",
	PENDING = "PENDING",
	RESOLVED = "RESOLVED",
	CLOSED = "CLOSED",
	CANCELED = "CANCELED",
}

export const StatusLabels: Record<StatusEnum, string> = {
	[StatusEnum.PENDING_APPROVAL]: "Aguardando Aprovação",
	[StatusEnum.NEW]: "Novo",
	[StatusEnum.IN_PROGRESS]: "Em Andamento",
	[StatusEnum.PENDING]: "Pendente",
	[StatusEnum.RESOLVED]: "Resolvido",
	[StatusEnum.CLOSED]: "Fechado",
	[StatusEnum.CANCELED]: "Cancelado",
};
